### DISCUSSION TITLE
Exporting notebook/html based on starting notebook after changing variables

### DISCUSSION THREAD

**Discussion by eford** [fonsp/Pluto.jl#1705] [Pluto development]
Is there a good way to:
1. open a template notebook,
2. 2.  change some variables (like the PlutoRESTClient server does), and
3. export the results to a new notebook (to a standard Pluto notebook and/or to html)?  

The `export_notebook` function doesn't seem to with with a PlutoNotebook object created from the PlutoRESTClient.

**Comment by seldomworks** [comment]
I think this would be extremely useful to me as well. In my head, the concept is similar to what papermill provides for jupyter notebooks.
